packages/peer-exchange/package.json

```diff
@@ -0,0 +1,106 @@
+{
+  "name": "@waku/peer-exchange",
+  "version": "0.0.1",
+  "description": "Peer Exchange (https://rfc.vac.dev/spec/34/) protocol for Waku",
+  "types": "./dist/index.d.ts",
+  "module": "./dist/index.js",
+  "exports": {
+    ".": {
+      "types": "./dist/index.d.ts",
+      "import": "./dist/index.js"
+    }
+  },
+  "type": "module",
+  "author": "Waku Team",
+  "homepage": "https://github.com/waku-org/js-waku/tree/master/packages/enr#readme",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/waku-org/js-waku.git"
+  },
+  "bugs": {
+    "url": "https://github.com/waku-org/js-waku/issues"
+  },
+  "license": "MIT OR Apache-2.0",
+  "keywords": [
+    "waku",
+    "decentralized",
+    "secure",
+    "communication",
+    "web3",
+    "ethereum",
+    "dapps",
+    "privacy"
+  ],
+  "scripts": {
+    "build": "run-s build:**",
+    "build:esm": "tsc",
+    "build:bundle": "rollup --config rollup.config.js",
+    "fix": "run-s fix:*",
+    "fix:prettier": "prettier . --write",
+    "fix:lint": "eslint src --ext .ts --ext .cjs --fix",
+    "check": "run-s check:*",
+    "check:lint": "eslint src --ext .ts",
+    "check:prettier": "prettier . --list-different",
+    "check:spelling": "cspell \"{README.md,src/**/*.ts}\"",
+    "check:tsc": "tsc -p tsconfig.dev.json",
+    "test": "run-s test:*",
+    "test:node": "TS_NODE_PROJECT=./tsconfig.dev.json mocha",
+    "test:browser": "karma start karma.conf.cjs",
+    "prepublish": "npm run build",
+    "reset-hard": "git clean -dfx -e .idea && git reset --hard && npm i && npm run build"
+  },
+  "engines": {
+    "node": ">=16"
+  },
+  "dependencies": {
+    "@libp2p/interface-connection": "^3.0.3",
+    "@libp2p/interface-peer-discovery": "^1.0.0",
+    "@libp2p/interface-peer-discovery-compliance-tests": "^2.0.1",
+    "@libp2p/interface-peer-id": "^1.0.6",
+    "@libp2p/interface-peer-info": "^1.0.4",
+    "@libp2p/interface-peer-store": "^1.2.3",
+    "@waku/core": "*",
```

Comment:
FYI https://github.com/waku-org/js-waku/issues/994 so we can remove this dep (less dep means less releasing due to dep)

Reply:
I can follow up with solving this: https://github.com/waku-org/js-waku/issues/1043